manifests/params.pp

```diff
@@ -255,6 +255,7 @@
   $server_startpollersunreachable           = '1'
   $server_startpreprocessors                = 3
   $server_startproxypollers                 = '1'
+  $server_startreportwriters                = undef
```

Comment:
please don't define the values here, but directly in the server.pp

Reply:
Gladly. I was just following every other param.